firefox_puppeteer/ui/toolbars.py

```diff
@@ -446,3 +446,21 @@ def verifier(self):
         :returns: Reference to the identity-popup content verifier.
         """
         return self.marionette.find_element(By.ID, 'identity-popup-content-verifier')
+
+    def close(self, force=False):
+        """Closes the identity popup by hitting the escape key.
+
+        :param force: Optional, if True, the popup will be force closed,
+         default to `False`.
+        """
+        if expected.element_not_present(By.ID, 'identity-popup') or not self.is_open:
```

Comment:
The panel does always exist. It's not getting removed from the DOM. So the expected check should really not be necessary. Do you have some example code I can execute to see that?